conans/client/client_cache.py

```diff
@@ -29,6 +29,7 @@
 REGISTRY_JSON = "registry.json"
 PROFILES_FOLDER = "profiles"
 HOOKS_FOLDER = "hooks"
+EDITABLES_FILE = "editable_packages.ini"
```

Comment:
Could be named similar to the repository file? ``conan_edit_layout``? maybe avoid the ``.ini`` extension? I would expect some analogy in name and syntax in both places.

Reply:
We don't know where this file information will be. We have talked about a _profile-like_ file, but we need to define this. Move this conversation to the main PR.